<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main</class>
 <widget class="QWidget" name="main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>main</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(232, 232, 232)</string>
  </property>
  <widget class="QLabel" name="label_image">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>90</y>
     <width>231</width>
     <height>211</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>C:/Users/bentu/OneDrive/Masaüstü/photo.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_port">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>90</y>
     <width>231</width>
     <height>211</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>C:/Users/bentu/OneDrive/Masaüstü/port-150928_1280.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>360</y>
     <width>221</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_serial">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>310</y>
     <width>251</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Wählen Sie eine serielle Schnittstelle</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_tau">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>10</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_take">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>430</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none;</string>
   </property>
   <property name="text">
    <string>Mach ein Foto</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_run1">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>430</y>
     <width>121</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none;</string>
   </property>
   <property name="text">
    <string>Zeichne dieses Bild</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_photo">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>350</y>
     <width>121</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_run">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>360</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/icons/a.png);
border:none</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="imgLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>371</width>
     <height>341</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Back">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>460</y>
     <width>121</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_take2">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>450</y>
     <width>121</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>530</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Mach ein Foto</string>
   </property>
  </widget>
  <widget class="QLabel" name="label2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>530</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Zurück</string>
   </property>
  </widget>
  <widget class="QLabel" name="imgLabel_2">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>90</y>
     <width>371</width>
     <height>341</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_warning">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>160</y>
     <width>391</width>
     <height>191</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 5px solid black; background: white; color: black;</string>
   </property>
   <property name="text">
    <string>Ihre Transaktion läuft, bitte warten Sie ...</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>290</y>
     <width>191</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QProgressBar::chunk {
    background-color: #05B8CC;
    
}</string>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
